prov/efa/src/rdm/rxr_ep.c

```diff
@@ -1040,6 +1040,19 @@ static int rxr_ep_setopt(fid_t fid, int level, int optname,
 		if (ret)
 			return ret;
 		break;
+	case FI_OPT_EFA_SEND_IN_ORDER_128_BYTES:
+		/* For future use only, currently we do not support in-order 
+		transfer in send/recv */
+		return -FI_EOPNOTSUPP;
+	case FI_OPT_EFA_RMA_IN_ORDER_128_BYTES:
```

Comment:
I did not notice `ibv_query_qp_data_in_order` work on op. so this should be `FI_OPT_EFA_WRITE_IN_ORDER_128_BYTES

Reply:
Also added `ALIGNED`